scripts/docker-entrypoint.sh

```diff
@@ -305,6 +330,13 @@ fi
 if [ -e "${server_xml}" ]; then
   echo "Loading server.xml from ${server_xml}...";
   cp "${server_xml}" "${CATALINA_HOME}/conf/"
+elif [ -e "${config_root}/server.xml.tmpl" ]; then
+  #server.xml.tmpl
+  echo "No server.xml was specified in ${server_xml}.  Generating from templates."
+  if [ -e ${config_root}/server.xml.tmpl ] ; then
+    cp ${config_root}/server.xml.tmpl ${CATALINA_HOME}/conf/server.xml.tmpl
+  fi
```

Comment:
I guess above condition is duplicate. Can this be removed?

Reply:
removing that